cpp/tests/datetime/datetime_ops_test.cpp

```diff
@@ -827,4 +843,76 @@ TEST_F(BasicDatetimeOpsTest, TestQuarter)
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(*extract_quarter(timestamps_s), quarter);
 }
 
+TYPED_TEST(TypedDatetimeOpsTest, TestFloorDatetime)
+{
+  using T = TypeParam;
+  using namespace cudf::test;
+  using namespace cudf::datetime;
+  using namespace cuda::std::chrono;
+
+  auto start = milliseconds(-2500000000000);  // Sat, 11 Oct 1890 19:33:20 GMT
+  auto stop_ = milliseconds(2500000000000);   // Mon, 22 Mar 2049 04:26:40 GMT
```

Comment:
Is there a name conflict that requires the use of `stop_`? If not, I would prefer `stop` to unify the name conventions. Also applies to `ceil` tests above.
```suggestion
  auto start = milliseconds(-2500000000000);  // Sat, 11 Oct 1890 19:33:20 GMT
  auto stop  = milliseconds(2500000000000);   // Mon, 22 Mar 2049 04:26:40 GMT
```

Reply:
The other tests in this file like `TestExtractingGeneratedDatetimeComponents` and `TestExtractingGeneratedNullableDatetimeComponents` also use `stop_` as a convention, should we change them as well then?

Reply:
I would leave changes to other tests to a future refactor.